conda_smithy/templates/github-actions.tmpl

```diff
@@ -198,8 +193,14 @@ jobs:
         if EXIST LICENSE.txt (
           copy LICENSE.txt "{{ recipe_dir }}\\recipe-scripts-license.txt"
         )
-        {%- if build_with_mambabuild %}
+        {%- if conda_build_tool == "mambabuild" %}
         conda.exe mambabuild "{{ recipe_dir }}" -m .ci_support\%CONFIG%.yaml
+        {%- elif conda_build_tool == "conda-build+conda-libmamba-solver" %}
+        set "CONDA_SOLVER=libmamba"
+        conda.exe build "{{ recipe_dir }}" -m .ci_support\%CONFIG%.yaml
+        {%- elif conda_build_tool == "conda-build+classic" %}
+        set "CONDA_SOLVER=classic"
+        conda.exe build "{{ recipe_dir }}" -m .ci_support\%CONFIG%.yaml
```

Comment:
This bit of the template is a bit repetitive but I'm not gonna be picky about it.

Reply:
Yep I don't like it either, but nested ifs in YAML are weird given how we can't indent things for visual cues. The obvious refactor here is to take the Windows logic into a .bat file like we do on Linux and macOS, though. So at least the repetition is in a single place, and not in two 😬 

The other option is to create a Jinja function that prints the right command on the template, and handle the logic in Python land.

Reply:
nah no worries here - seems like it is too much of a pain to fix

Reply:
Added https://github.com/conda-forge/conda-smithy/pull/1761 to handle this kind of thing better in the future.